app/package.json

```diff
@@ -31,7 +31,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "eject": "react-scripts eject",
-    "gql-gen": "graphql-codegen --config codegen.js --require dotenv/config"
+    "gql-gen": "graphql-codegen --config codegen.js --require dotenv/config",
+    "lint": "eslint --ext ts,tsx src && prettier --check \"src/{**/*,*}.{js,ts,jsx,tsx}\""
```

Comment:
`prettier` is failing in its current state, which we can fix assuming we want to use it?

We could also split these up into separate scripts / jobs for clarity?